<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Introduction to regionReport • regionReport</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Introduction to regionReport">
<meta property="og:description" content="regionReport">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">regionReport</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.35.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/regionReport.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/bumphunterExample.html">Example report using bumphunter results</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/leekgroup/regionReport/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Introduction to regionReport</h1>
                        <h4 data-toc-skip class="author">Leonardo
Collado-Torres</h4>
            <address class="author_afil">
      Lieber Institute for Brain Development, Johns Hopkins Medical
CampusCenter for Computational Biology, Johns Hopkins
University<br><a class="author_email" href="mailto:#"></a><a href="mailto:lcolladotor@gmail.com" class="email">lcolladotor@gmail.com</a>
      </address>
                  
            <h4 data-toc-skip class="date">7 May 2023</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/leekgroup/regionReport/blob/HEAD/vignettes/regionReport.Rmd" class="external-link"><code>vignettes/regionReport.Rmd</code></a></small>
      <div class="hidden name"><code>regionReport.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="basics">Basics<a class="anchor" aria-label="anchor" href="#basics"></a>
</h2>
<div class="section level3">
<h3 id="install-regionreport">Install <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em><a class="anchor" aria-label="anchor" href="#install-regionreport"></a>
</h3>
<p><code>R</code> is an open-source statistical environment which can be
easily modified to enhance its functionality via packages. <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
is a <code>R</code> package available via the <a href="http://bioconductor/packages/regionReport" class="external-link">Bioconductor</a>
repository for packages. <code>R</code> can be installed on any
operating system from <a href="https://cran.r-project.org/" class="external-link">CRAN</a>
after which you can install <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
by using the following commands in your <code>R</code> session:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/ns-load.html" class="external-link">requireNamespace</a></span><span class="op">(</span><span class="st">"BiocManager"</span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span> <span class="op">{</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span></span>
<span><span class="op">}</span></span>
<span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/BiocManager/man/install.html" class="external-link">install</a></span><span class="op">(</span><span class="st">"regionReport"</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## Check that you have a valid Bioconductor installation</span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/BiocManager/man/valid.html" class="external-link">valid</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="required-knowledge">Required knowledge<a class="anchor" aria-label="anchor" href="#required-knowledge"></a>
</h3>
<p><em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
is based on many other packages and in particular in those that have
implemented the infrastructure needed for dealing with RNA-seq data.
That is, packages like <em><a href="https://bioconductor.org/packages/3.17/GenomicFeatures" class="external-link">GenomicFeatures</a></em>
that allow you to import the data and <em><a href="https://bioconductor.org/packages/3.17/DESeq2" class="external-link">DESeq2</a></em> for
generating differential expression results. A <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
user is not expected to deal with those packages directly.</p>
<p>If you are asking yourself the question “Where do I start using
Bioconductor?” you might be interested in <a href="http://lcolladotor.github.io/2014/10/16/startbioc/#.VkOKbq6rRuU" class="external-link">this
blog post</a>.</p>
</div>
<div class="section level3">
<h3 id="asking-for-help">Asking for help<a class="anchor" aria-label="anchor" href="#asking-for-help"></a>
</h3>
<p>As package developers, we try to explain clearly how to use our
packages and in which order to use the functions. But <code>R</code> and
<code>Bioconductor</code> have a steep learning curve so it is critical
to learn where to ask for help. The blog post quoted above mentions some
but we would like to highlight the <a href="https://support.bioconductor.org/" class="external-link">Bioconductor support site</a>
as the main resource for getting help: remember to use the
<code>regionReport</code> tag and check <a href="https://support.bioconductor.org/t/regionReport/" class="external-link">the older
posts</a>. Other alternatives are available such as creating GitHub
issues and tweeting. However, please note that if you want to receive
help you should adhere to the <a href="http://www.bioconductor.org/help/support/posting-guide/" class="external-link">posting
guidelines</a>. It is particularly critical that you provide a small
reproducible example and your session information so package developers
can track down the source of the error.</p>
</div>
<div class="section level3">
<h3 id="citing-regionreport">Citing <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em><a class="anchor" aria-label="anchor" href="#citing-regionreport"></a>
</h3>
<p>We hope that <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
will be useful for your research. Please use the following information
to cite the package and the overall approach. Thank you!</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Citation info</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/citation.html" class="external-link">citation</a></span><span class="op">(</span><span class="st">"regionReport"</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## To cite package 'regionReport' in publications use:</span></span>
<span><span class="co">## </span></span>
<span><span class="co">##   Collado-Torres L, Jaffe AE, Leek JT (2016). "regionReport:</span></span>
<span><span class="co">##   Interactive reports for region-level and feature-level genomic</span></span>
<span><span class="co">##   analyses [version2; referees: 2 approved, 1 approved with</span></span>
<span><span class="co">##   reservations]." _F1000Research_, *4*, 105.</span></span>
<span><span class="co">##   doi:10.12688/f1000research.6379.2</span></span>
<span><span class="co">##   &lt;https://doi.org/10.12688/f1000research.6379.2&gt;,</span></span>
<span><span class="co">##   &lt;http://f1000research.com/articles/4-105/v2&gt;.</span></span>
<span><span class="co">## </span></span>
<span><span class="co">##   Collado-Torres L, Nellore A, Frazee AC, Wilks C, Love MI, Langmead B,</span></span>
<span><span class="co">##   Irizarry RA, Leek JT, Jaffe AE (2017). "Flexible expressed region</span></span>
<span><span class="co">##   analysis for RNA-seq with derfinder." _Nucl. Acids Res._.</span></span>
<span><span class="co">##   doi:10.1093/nar/gkw852 &lt;https://doi.org/10.1093/nar/gkw852&gt;,</span></span>
<span><span class="co">##   &lt;http://nar.oxfordjournals.org/content/early/2016/09/29/nar.gkw852&gt;.</span></span>
<span><span class="co">## </span></span>
<span><span class="co">##   Collado-Torres L, Jaffe AE, Leek JT (2017). _regionReport: Generate</span></span>
<span><span class="co">##   HTML or PDF reports for a set of genomic regions or DESeq2/edgeR</span></span>
<span><span class="co">##   results_. doi:10.18129/B9.bioc.regionReport</span></span>
<span><span class="co">##   &lt;https://doi.org/10.18129/B9.bioc.regionReport&gt;,</span></span>
<span><span class="co">##   https://github.com/leekgroup/regionReport - R package version 1.35.0,</span></span>
<span><span class="co">##   &lt;http://www.bioconductor.org/packages/regionReport&gt;.</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## To see these entries in BibTeX format, use 'print(&lt;citation&gt;,</span></span>
<span><span class="co">## bibtex=TRUE)', 'toBibtex(.)', or set</span></span>
<span><span class="co">## 'options(citation.bibtex.max=999)'.</span></span></code></pre>
</div>
</div>
<div class="section level2">
<h2 id="html-reports-for-a-set-differential-region-results">HTML reports for a set differential region results<a class="anchor" aria-label="anchor" href="#html-reports-for-a-set-differential-region-results"></a>
</h2>
<p><em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
(Collado-Torres, Jaffe, and Leek, 2016) creates HTML or PDF reports for
a set of genomic regions such as <em><a href="https://bioconductor.org/packages/3.17/derfinder" class="external-link">derfinder</a></em>
(Collado-Torres, Nellore, Frazee, Wilks, Love, Langmead, Irizarry, Leek,
and Jaffe, 2017) results or for feature-level analyses performed with
<em><a href="https://bioconductor.org/packages/3.17/DESeq2" class="external-link">DESeq2</a></em>
(Love, Huber, and Anders, 2014) or <em><a href="https://bioconductor.org/packages/3.17/edgeR" class="external-link">edgeR</a></em>
(Robinson, McCarthy, and Smyth, 2010; McCarthy, Chen, and Smyth, 2012;
Chen, Lun, and Smyth, 2014). The HTML reports are styled with <em><a href="https://CRAN.R-project.org/package=rmarkdown" class="external-link">rmarkdown</a></em>
(Allaire, Xie, Dervieux, McPherson, Luraschi, Ushey, Atkins, Wickham,
Cheng, Chang, and Iannone, 2023) by default but can optionally be styled
with <em><a href="https://CRAN.R-project.org/package=knitrBootstrap" class="external-link">knitrBootstrap</a></em>
(Hester, 2018).</p>
<p>This package includes a basic exploration for a general set of
genomic regions which can be easily customized to include the
appropriate conclusions and/or further exploration of the results. Such
a report can be generated using <code><a href="../reference/renderReport.html">renderReport()</a></code>. <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
has a separate template for running a basic exploration analysis of
<em><a href="https://bioconductor.org/packages/3.17/derfinder" class="external-link">derfinder</a></em>
results by using <code><a href="../reference/derfinderReport.html">derfinderReport()</a></code>.
<code><a href="../reference/derfinderReport.html">derfinderReport()</a></code> is specific to single base-level approach
<em><a href="https://bioconductor.org/packages/3.17/derfinder" class="external-link">derfinder</a></em>
results. A third template is included for exploring <em><a href="https://bioconductor.org/packages/3.17/DESeq2" class="external-link">DESeq2</a></em> or
<em><a href="https://bioconductor.org/packages/3.17/edgeR" class="external-link">edgeR</a></em>
differential expression results.</p>
<p>All reports are written in <a href="http://rmarkdown.rstudio.com/" class="external-link">R
Markdown</a> format and include all the code for making the plots and
explorations in the report itself. For all templates, <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
relies on <em><a href="https://CRAN.R-project.org/package=knitr" class="external-link">knitr</a></em> (Xie,
2014), <em><a href="https://CRAN.R-project.org/package=rmarkdown" class="external-link">rmarkdown</a></em>
(Allaire, Xie, Dervieux et al., 2023), <code>DT</code> (Xie, Cheng, and
Tan, 2023) and optionally <em><a href="https://CRAN.R-project.org/package=knitrBootstrap" class="external-link">knitrBootstrap</a></em>
(Hester, 2018) for generating the report. The reports can be either in
HTML or PDF format and can be easily customized.</p>
</div>
<div class="section level2">
<h2 id="using-regionreport-for-deseq2-results">Using <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
for <em><a href="https://bioconductor.org/packages/3.17/DESeq2" class="external-link">DESeq2</a></em>
results<a class="anchor" aria-label="anchor" href="#using-regionreport-for-deseq2-results"></a>
</h2>
<p>The plots in <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
for exploring <em><a href="https://bioconductor.org/packages/3.17/DESeq2" class="external-link">DESeq2</a></em> are
powered by <em><a href="https://CRAN.R-project.org/package=ggplot2" class="external-link">ggplot2</a></em>
(Wickham, 2016) and <em><a href="https://CRAN.R-project.org/package=pheatmap" class="external-link">pheatmap</a></em>
(Kolde, 2019).</p>
<div class="section level3">
<h3 id="example">Example<a class="anchor" aria-label="anchor" href="#example"></a>
</h3>
<p>The <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
supplementary website <a href="http://leekgroup.github.io/regionReportSupp/" class="external-link">regionReportSupp</a>
has examples of using <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
with <em><a href="https://bioconductor.org/packages/3.17/DESeq2" class="external-link">DESeq2</a></em>
results. In particular, please look at <a href="http://leekgroup.github.io/regionReportSupp/DESeq2.html" class="external-link">DESeq2.html</a>
which has the code for generating some <em><a href="https://bioconductor.org/packages/3.17/DESeq2" class="external-link">DESeq2</a></em>
results based on the <em><a href="https://bioconductor.org/packages/3.17/DESeq2" class="external-link">DESeq2</a></em>
vignette. Then it uses those results to create HTML and PDF versions of
the same report. The resulting reports are available in the following
locations:</p>
<ul>
<li><a href="http://leekgroup.github.io/regionReportSupp/DESeq2-example/index.html" class="external-link">HTML
version</a></li>
<li><a href="http://leekgroup.github.io/regionReportSupp/DESeq2-example/DESeq2Report.pdf" class="external-link">PDF
version</a></li>
</ul>
<p>Note that in both examples we changed the <em><a href="https://CRAN.R-project.org/package=ggplot2" class="external-link">ggplot2</a></em> theme
to <code>theme_bw()</code>. Also, in the PDF version we used the option
<code>device = 'pdf'</code> instead of the default
<code>device = 'png'</code> in <code><a href="../reference/DESeq2Report.html">DESeq2Report()</a></code> since PDF
figures are more appropriate for PDF reports: they look better than PNG
figures.</p>
<p>If you want to create a similar HTML report as the one linked in this
section, simply run
<code>example('DESeq2Report', 'regionReport', ask=FALSE)</code>. The
only difference will be the <em><a href="https://CRAN.R-project.org/package=ggplot2" class="external-link">ggplot2</a></em> theme
for the plots.</p>
</div>
</div>
<div class="section level2">
<h2 id="using-regionreport-for-edger-results">Using <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
for <em><a href="https://bioconductor.org/packages/3.17/edgeR" class="external-link">edgeR</a></em>
results<a class="anchor" aria-label="anchor" href="#using-regionreport-for-edger-results"></a>
</h2>
<p><em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
has the <code><a href="../reference/edgeReport.html">edgeReport()</a></code> function that takes as input a
<code>DGEList</code> object and the results from the differential
expression analysis using <em><a href="https://bioconductor.org/packages/3.17/edgeR" class="external-link">edgeR</a></em>.
<code><a href="../reference/edgeReport.html">edgeReport()</a></code> internally uses <em><a href="https://bioconductor.org/packages/3.17/DEFormats" class="external-link">DEFormats</a></em>
to convert the results to <em><a href="https://bioconductor.org/packages/3.17/DESeq2" class="external-link">DESeq2</a></em>’s
format and then uses <code>DESeqReport()</code> to create the final
report. The report looks nearly the same whether you performed the
differential expression analysis with <em><a href="https://bioconductor.org/packages/3.17/DESeq2" class="external-link">DESeq2</a></em> or
<em><a href="https://bioconductor.org/packages/3.17/edgeR" class="external-link">edgeR</a></em> in
order to make more homogenous the exploratory data analysis step.</p>
<div class="section level3">
<h3 id="example-1">Example<a class="anchor" aria-label="anchor" href="#example-1"></a>
</h3>
<p>The <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
supplementary website <a href="http://leekgroup.github.io/regionReportSupp/" class="external-link">regionReportSupp</a>
has examples of using <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
with <em><a href="https://bioconductor.org/packages/3.17/edgeR" class="external-link">edgeR</a></em>
results. In particular, please look at <a href="http://leekgroup.github.io/regionReportSupp/edgeR.html" class="external-link">edgeR.html</a>
which has the code for generating some random data with <em><a href="https://bioconductor.org/packages/3.17/DEFormats" class="external-link">DEFormats</a></em>
and performing the differential expression analysis with <em><a href="https://bioconductor.org/packages/3.17/edgeR" class="external-link">edgeR</a></em>. Then
it uses those results to create HTML and PDF versions of the same
report. The resulting reports are available in the following
locations:</p>
<ul>
<li><a href="http://leekgroup.github.io/regionReportSupp/edgeR-example/index.html" class="external-link">HTML
version</a></li>
<li><a href="http://leekgroup.github.io/regionReportSupp/edgeR-example/edgeReport.pdf" class="external-link">PDF
version</a></li>
</ul>
<p>Note that in both examples we changed the <em><a href="https://CRAN.R-project.org/package=ggplot2" class="external-link">ggplot2</a></em> theme
to <code>theme_linedraw()</code>. Also, in the PDF version we used the
option <code>device = 'pdf'</code> instead of the default
<code>device = 'png'</code> in <code><a href="../reference/edgeReport.html">edgeReport()</a></code> since PDF
figures are more appropriate for PDF reports: they look better than PNG
figures.</p>
<p>If you want to create a similar HTML report as the one linked in this
section, simply run
<code>example('edgeReport', 'regionReport', ask=FALSE)</code>. The only
difference will be the <em><a href="https://CRAN.R-project.org/package=ggplot2" class="external-link">ggplot2</a></em> theme
for the plots and the amount of data simulated with <em><a href="https://bioconductor.org/packages/3.17/DEFormats" class="external-link">DEFormats</a></em>.</p>
</div>
</div>
<div class="section level2">
<h2 id="using-regionreport-for-region-results">Using <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
for region results<a class="anchor" aria-label="anchor" href="#using-regionreport-for-region-results"></a>
</h2>
<p>The plots in <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
for region reports are powered by <em><a href="https://bioconductor.org/packages/3.17/derfinderPlot" class="external-link">derfinderPlot</a></em>
(Collado-Torres, Jaffe, and Leek, 2017), <em><a href="https://bioconductor.org/packages/3.17/ggbio" class="external-link">ggbio</a></em> (Yin,
Cook, and Lawrence, 2012), and <em><a href="https://CRAN.R-project.org/package=ggplot2" class="external-link">ggplot2</a></em>
(Wickham, 2016).</p>
<div class="section level3">
<h3 id="examples">Examples<a class="anchor" aria-label="anchor" href="#examples"></a>
</h3>
<p>The <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
supplementary website <a href="http://leekgroup.github.io/regionReportSupp/" class="external-link">regionReportSupp</a>
has examples of using <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
with results from <em><a href="https://bioconductor.org/packages/3.17/DiffBind" class="external-link">DiffBind</a></em>
and <em><a href="https://bioconductor.org/packages/3.17/derfinder" class="external-link">derfinder</a></em>.
Included as a vignette, this package also has an example using a small
data set derived from <em><a href="https://bioconductor.org/packages/3.17/bumphunter" class="external-link">bumphunter</a></em>.
These represent different uses of <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>
for results from ChIP-seq, methylation, and RNA-seq data. In particular,
the <em><a href="https://bioconductor.org/packages/3.17/DiffBind" class="external-link">DiffBind</a></em>
example illustrates how to expand a basic report created with
<code><a href="../reference/renderReport.html">renderReport()</a></code>.</p>
</div>
<div class="section level3">
<h3 id="general-case">General case<a class="anchor" aria-label="anchor" href="#general-case"></a>
</h3>
<p>For a general use case, you first have to identify a set of genomic
regions of interest and store it as a <code>GRanges</code> object. In a
typical workflow you will have some variables measured for each of the
regions, such as p-values and scores. <code><a href="../reference/renderReport.html">renderReport()</a></code> uses
the set of regions and three main arguments:</p>
<ul>
<li>
<code>pvalueVars</code>: this is a character vector (named
optionally) with the names of the variables that are bound between 0 and
1, such as p-values. For each of these variables,
<code><a href="../reference/renderReport.html">renderReport()</a></code> explores the distribution by chromosome, the
overall distribution, and makes a table with commonly used cutoffs.</li>
<li>
<code>densityVars</code>: is another character vector (named
optionally) with another set of variables you wish to explore by making
density graphs. This is commonly used for scores and other similar
numerical variables.</li>
<li>
<code>significantVar</code>: is a logical vector separating the
regions into by whether they are statistically significant. For example,
this information is used to explore the width of all the regions and
compare it the significant ones.</li>
</ul>
<p>Other parameters control the name of the report, where it’ll be
located, the transcripts database used to annotate the nearest genes,
graphical parameters, etc.</p>
<p>Here is a short example of how to use <code><a href="../reference/renderReport.html">renderReport()</a></code>.
Note that we are using regions produced by <em><a href="https://bioconductor.org/packages/3.17/derfinder" class="external-link">derfinder</a></em>
just for convenience sake.</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Load derfinder</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="st"><a href="https://github.com/lcolladotor/derfinder" class="external-link">"derfinder"</a></span><span class="op">)</span></span>
<span><span class="va">regions</span> <span class="op">&lt;-</span> <span class="va">genomeRegions</span><span class="op">$</span><span class="va">regions</span></span>
<span></span>
<span><span class="co">## Assign chr length</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="st"><a href="https://bioconductor.org/packages/GenomicRanges" class="external-link">"GenomicRanges"</a></span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/GenomeInfoDb/man/seqinfo.html" class="external-link">seqlengths</a></span><span class="op">(</span><span class="va">regions</span><span class="op">)</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"chr21"</span> <span class="op">=</span> <span class="fl">48129895</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## The output will be saved in the 'derfinderReport-example' directory</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/files2.html" class="external-link">dir.create</a></span><span class="op">(</span><span class="st">"renderReport-example"</span>, showWarnings <span class="op">=</span> <span class="cn">FALSE</span>, recursive <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## Generate the HTML report</span></span>
<span><span class="va">report</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/renderReport.html">renderReport</a></span><span class="op">(</span><span class="va">regions</span>, <span class="st">"Example run"</span>,</span>
<span>    pvalueVars <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>        <span class="st">"Q-values"</span> <span class="op">=</span> <span class="st">"qvalues"</span>, <span class="st">"P-values"</span> <span class="op">=</span> <span class="st">"pvalues"</span></span>
<span>    <span class="op">)</span>, densityVars <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>        <span class="st">"Area"</span> <span class="op">=</span> <span class="st">"area"</span>, <span class="st">"Mean coverage"</span> <span class="op">=</span> <span class="st">"meanCoverage"</span></span>
<span>    <span class="op">)</span>,</span>
<span>    significantVar <span class="op">=</span> <span class="va">regions</span><span class="op">$</span><span class="va">qvalues</span> <span class="op">&lt;=</span> <span class="fl">0.05</span>, nBestRegions <span class="op">=</span> <span class="fl">20</span>,</span>
<span>    outdir <span class="op">=</span> <span class="st">"renderReport-example"</span></span>
<span><span class="op">)</span></span></code></pre></div>
<p>See the report created by this example <a href="http://leekgroup.github.io/regionReport/reference/renderReport-example/regionExploration.html" class="external-link">here</a>.</p>
<p>For <em><a href="https://bioconductor.org/packages/3.17/derfinder" class="external-link">derfinder</a></em>
results created via the expressed regions-level approach you can use
<code><a href="../reference/renderReport.html">renderReport()</a></code> to explore the results. If you use <em><a href="https://bioconductor.org/packages/3.17/DESeq2" class="external-link">DESeq2</a></em> to
perform the differential expression analysis of the expressed regions
you can then use <code><a href="../reference/DESeq2Report.html">DESeq2Report()</a></code>.</p>
</div>
<div class="section level3">
<h3 id="derfinder-single-base-level-case">
<em><a href="https://bioconductor.org/packages/3.17/derfinder" class="external-link">derfinder</a></em>
single base-level case<a class="anchor" aria-label="anchor" href="#derfinder-single-base-level-case"></a>
</h3>
<div class="section level4">
<h4 id="run-derfinder">Run <em><a href="https://bioconductor.org/packages/3.17/derfinder" class="external-link">derfinder</a></em><a class="anchor" aria-label="anchor" href="#run-derfinder"></a>
</h4>
<p>Prior to using <code><a href="../reference/derfinderReport.html">regionReport::derfinderReport()</a></code> you must
use <em><a href="https://bioconductor.org/packages/3.17/derfinder" class="external-link">derfinder</a></em>
to analyze a specific data set. While there are many ways to do so, we
recommend using <strong>analyzeChr()</strong> with the same
<em>prefix</em> argument. Then merging the results with
<strong>mergeResults()</strong>. This is the recommended pipeline for
the single base-level approach.</p>
<p>Below, we run <em><a href="https://bioconductor.org/packages/3.17/derfinder" class="external-link">derfinder</a></em>
for the example data included in the package. The steps are:</p>
<ol style="list-style-type: decimal">
<li>Load derfinder</li>
<li>Create a directory where we’ll store the results</li>
<li>Generate the pre-requisites for the models to use with the example
data</li>
<li>Generate the statistical models</li>
<li>Analyze the example data for chr21</li>
<li>Merge the results (only one chr in this case, but in practice
there’ll be more)</li>
</ol>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Load derfinder</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="st"><a href="https://github.com/lcolladotor/derfinder" class="external-link">"derfinder"</a></span><span class="op">)</span></span>
<span></span>
<span><span class="co">## The output will be saved in the "derfinderReport-example" directory</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/files2.html" class="external-link">dir.create</a></span><span class="op">(</span><span class="st">"derfinderReport-example"</span>, showWarnings <span class="op">=</span> <span class="cn">FALSE</span>, recursive <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
<p>The following code runs <em><a href="https://bioconductor.org/packages/3.17/derfinder" class="external-link">derfinder</a></em>.</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Save the current path</span></span>
<span><span class="va">initialPath</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/getwd.html" class="external-link">getwd</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/getwd.html" class="external-link">setwd</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/file.path.html" class="external-link">file.path</a></span><span class="op">(</span><span class="va">initialPath</span>, <span class="st">"derfinderReport-example"</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## Generate output from derfinder</span></span>
<span></span>
<span><span class="co">## Collapse the coverage information</span></span>
<span><span class="va">collapsedFull</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/derfinder/man/collapseFullCoverage.html" class="external-link">collapseFullCoverage</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="va">genomeData</span><span class="op">$</span><span class="va">coverage</span><span class="op">)</span>,</span>
<span>    verbose <span class="op">=</span> <span class="cn">TRUE</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co">## Calculate library size adjustments</span></span>
<span><span class="va">sampleDepths</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/derfinder/man/sampleDepth.html" class="external-link">sampleDepth</a></span><span class="op">(</span><span class="va">collapsedFull</span>,</span>
<span>    probs <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="fl">0.5</span><span class="op">)</span>, nonzero <span class="op">=</span> <span class="cn">TRUE</span>,</span>
<span>    verbose <span class="op">=</span> <span class="cn">TRUE</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co">## Build the models</span></span>
<span><span class="va">group</span> <span class="op">&lt;-</span> <span class="va">genomeInfo</span><span class="op">$</span><span class="va">pop</span></span>
<span><span class="va">adjustvars</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span><span class="va">genomeInfo</span><span class="op">$</span><span class="va">gender</span><span class="op">)</span></span>
<span><span class="va">models</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/derfinder/man/makeModels.html" class="external-link">makeModels</a></span><span class="op">(</span><span class="va">sampleDepths</span>, testvars <span class="op">=</span> <span class="va">group</span>, adjustvars <span class="op">=</span> <span class="va">adjustvars</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## Analyze chromosome 21</span></span>
<span><span class="va">analysis</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/derfinder/man/analyzeChr.html" class="external-link">analyzeChr</a></span><span class="op">(</span></span>
<span>    chr <span class="op">=</span> <span class="st">"21"</span>, coverageInfo <span class="op">=</span> <span class="va">genomeData</span>, models <span class="op">=</span> <span class="va">models</span>,</span>
<span>    cutoffFstat <span class="op">=</span> <span class="fl">1</span>, cutoffType <span class="op">=</span> <span class="st">"manual"</span>, seeds <span class="op">=</span> <span class="fl">20140330</span>, groupInfo <span class="op">=</span> <span class="va">group</span>,</span>
<span>    mc.cores <span class="op">=</span> <span class="fl">1</span>, writeOutput <span class="op">=</span> <span class="cn">TRUE</span>, returnOutput <span class="op">=</span> <span class="cn">TRUE</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co">## Save the stats options for later</span></span>
<span><span class="va">optionsStats</span> <span class="op">&lt;-</span> <span class="va">analysis</span><span class="op">$</span><span class="va">optionsStats</span></span>
<span></span>
<span><span class="co">## Change the directory back to the original one</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/getwd.html" class="external-link">setwd</a></span><span class="op">(</span><span class="va">initialPath</span><span class="op">)</span></span></code></pre></div>
<p>For convenience, we have included the <em><a href="https://bioconductor.org/packages/3.17/derfinder" class="external-link">derfinder</a></em>
results as part of <em><a href="https://bioconductor.org/packages/3.17/regionReport" class="external-link">regionReport</a></em>.
Note that the above functions are routinely checked as part of <em><a href="https://bioconductor.org/packages/3.17/derfinder" class="external-link">derfinder</a></em>.</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Copy previous results</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/files.html" class="external-link">file.copy</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/file.path.html" class="external-link">file.path</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"chr21"</span><span class="op">)</span>,</span>
<span>    package <span class="op">=</span> <span class="st">"derfinder"</span>,</span>
<span>    mustWork <span class="op">=</span> <span class="cn">TRUE</span></span>
<span><span class="op">)</span>, <span class="st">"derfinderReport-example"</span>, recursive <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] TRUE</span></span></code></pre>
<p>Next, proceed to merging the results.</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Merge the results from the different chromosomes. In this case, there's</span></span>
<span><span class="co">## only one: chr21</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/derfinder/man/mergeResults.html" class="external-link">mergeResults</a></span><span class="op">(</span></span>
<span>    chrs <span class="op">=</span> <span class="st">"chr21"</span>, prefix <span class="op">=</span> <span class="st">"derfinderReport-example"</span>,</span>
<span>    genomicState <span class="op">=</span> <span class="va">genomicState</span><span class="op">$</span><span class="va">fullGenome</span></span>
<span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## 2023-05-07 01:27:32.733382 mergeResults: Saving options used</span></span></code></pre>
<pre><code><span><span class="co">## 2023-05-07 01:27:32.735004 Loading chromosome chr21</span></span></code></pre>
<pre><code><span><span class="co">## Neither 'cutoffFstatUsed' nor 'optionsStats' were supplied, so the FWER calculation step will be skipped.</span></span></code></pre>
<pre><code><span><span class="co">## 2023-05-07 01:27:32.868888 mergeResults: Saving fullNullSummary</span></span></code></pre>
<pre><code><span><span class="co">## 2023-05-07 01:27:32.869506 mergeResults: Re-calculating the p-values</span></span></code></pre>
<pre><code><span><span class="co">## 2023-05-07 01:27:32.999119 mergeResults: Saving fullRegions</span></span></code></pre>
<pre><code><span><span class="co">## 2023-05-07 01:27:33.000199 mergeResults: assigning genomic states</span></span></code></pre>
<pre><code><span><span class="co">## 2023-05-07 01:27:33.134913 annotateRegions: counting</span></span></code></pre>
<pre><code><span><span class="co">## 2023-05-07 01:27:33.217674 annotateRegions: annotating</span></span></code></pre>
<pre><code><span><span class="co">## 2023-05-07 01:27:33.24664 mergeResults: Saving fullAnnotatedRegions</span></span></code></pre>
<pre><code><span><span class="co">## 2023-05-07 01:27:33.247611 mergeResults: Saving fullFstats</span></span></code></pre>
<pre><code><span><span class="co">## 2023-05-07 01:27:33.248316 mergeResults: Saving fullTime</span></span></code></pre>
<div class="sourceCode" id="cb22"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Load optionsStats</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/load.html" class="external-link">load</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/file.path.html" class="external-link">file.path</a></span><span class="op">(</span><span class="st">"derfinderReport-example"</span>, <span class="st">"chr21"</span>, <span class="st">"optionsStats.Rdata"</span><span class="op">)</span>, verbose <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Loading objects:</span></span>
<span><span class="co">##   optionsStats</span></span></code></pre>
</div>
<div class="section level4">
<h4 id="create-report">Create report<a class="anchor" aria-label="anchor" href="#create-report"></a>
</h4>
<p>Once the <em><a href="https://bioconductor.org/packages/3.17/derfinder" class="external-link">derfinder</a></em>
output has been generated and merged, use
<strong>derfinderReport()</strong> to create the HTML report.</p>
<div class="sourceCode" id="cb24"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Load derfindeReport</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="st"><a href="https://github.com/leekgroup/regionReport" class="external-link">"regionReport"</a></span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb25"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Generate the HTML report</span></span>
<span><span class="va">report</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/derfinderReport.html">derfinderReport</a></span><span class="op">(</span></span>
<span>    prefix <span class="op">=</span> <span class="st">"derfinderReport-example"</span>, browse <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>    nBestRegions <span class="op">=</span> <span class="fl">15</span>, makeBestClusters <span class="op">=</span> <span class="cn">TRUE</span>,</span>
<span>    fullCov <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="st">"21"</span> <span class="op">=</span> <span class="va">genomeDataRaw</span><span class="op">$</span><span class="va">coverage</span><span class="op">)</span>, optionsStats <span class="op">=</span> <span class="va">optionsStats</span></span>
<span><span class="op">)</span></span></code></pre></div>
<p>Once the output is generated, you can browse the report from
<code>R</code> using <strong>browseURL()</strong> as shown below.</p>
<div class="sourceCode" id="cb26"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Browse the report</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/browseURL.html" class="external-link">browseURL</a></span><span class="op">(</span><span class="va">report</span><span class="op">)</span></span></code></pre></div>
<p>You can view a pre-compiled version of this report <a href="http://leekgroup.github.io/regionReport/reference/derfinderReport-example/basicExploration/basicExploration.html" class="external-link">here</a>.</p>
</div>
<div class="section level4">
<h4 id="notes">Notes<a class="anchor" aria-label="anchor" href="#notes"></a>
</h4>
<p>Note that the reports require an active Internet connection to render
correctly.</p>
<p>The report is self-explanatory and will change some of the text
depending on the input options.</p>
<p>If the report is taking too long to compile (say more than 3 hours),
you might want to consider setting <em>nBestCluters</em> to a small
number or even set <em>makeBestClusters</em> to <code>FALSE</code>.</p>
</div>
</div>
</div>
<div class="section level2">
<h2 id="reproducibility">Reproducibility<a class="anchor" aria-label="anchor" href="#reproducibility"></a>
</h2>
<p>This package was made possible thanks to:</p>
<ul>
<li>R (R Core Team, 2023)</li>
<li>
<em><a href="https://bioconductor.org/packages/3.17/BiocStyle" class="external-link">BiocStyle</a></em>
(Oleś, 2023)</li>
<li>
<em><a href="https://bioconductor.org/packages/3.17/biovizBase" class="external-link">biovizBase</a></em>
(Yin, Lawrence, and Cook, 2023)</li>
<li>
<em><a href="https://bioconductor.org/packages/3.17/bumphunter" class="external-link">bumphunter</a></em>
(Jaffe, Murakami, Lee, Leek, Fallin, Feinberg, and Irizarry, 2012)</li>
<li>
<em><a href="https://bioconductor.org/packages/3.17/DEFormats" class="external-link">DEFormats</a></em>
(Oleś, 2023)</li>
<li>
<em><a href="https://bioconductor.org/packages/3.17/derfinder" class="external-link">derfinder</a></em>
(Collado-Torres, Nellore, Frazee et al., 2017)</li>
<li>
<em><a href="https://bioconductor.org/packages/3.17/derfinderPlot" class="external-link">derfinderPlot</a></em>
(Collado-Torres, Jaffe, and Leek, 2017)</li>
<li>
<em><a href="https://bioconductor.org/packages/3.17/DESeq2" class="external-link">DESeq2</a></em>
(Love, Huber, and Anders, 2014)</li>
<li>
<em><a href="https://CRAN.R-project.org/package=DT" class="external-link">DT</a></em>
(Xie, Cheng, and Tan, 2023)</li>
<li>
<em><a href="https://bioconductor.org/packages/3.17/edgeR" class="external-link">edgeR</a></em>
(Robinson, McCarthy, and Smyth, 2010; McCarthy, Chen, and Smyth, 2012;
Chen, Lun, and Smyth, 2014)</li>
<li>
<em><a href="https://bioconductor.org/packages/3.17/GenomeInfoDb" class="external-link">GenomeInfoDb</a></em>
(Arora, Morgan, Carlson, and Pagès, 2017)</li>
<li>
<em><a href="https://bioconductor.org/packages/3.17/GenomicRanges" class="external-link">GenomicRanges</a></em>
(Lawrence, Huber, Pagès, Aboyoun, Carlson, Gentleman, Morgan, and Carey,
2013)</li>
<li>
<em><a href="https://bioconductor.org/packages/3.17/ggbio" class="external-link">ggbio</a></em> (Yin,
Cook, and Lawrence, 2012)</li>
<li>
<em><a href="https://CRAN.R-project.org/package=ggplot2" class="external-link">ggplot2</a></em>
(Wickham, 2016)</li>
<li>
<em><a href="https://CRAN.R-project.org/package=grid" class="external-link">grid</a></em>
(R Core Team, 2023)</li>
<li>
<em><a href="https://CRAN.R-project.org/package=gridExtra" class="external-link">gridExtra</a></em>
(Auguie, 2017)</li>
<li>
<em><a href="https://bioconductor.org/packages/3.17/IRanges" class="external-link">IRanges</a></em>
(Lawrence, Huber, Pagès et al., 2013)</li>
<li>
<em><a href="https://CRAN.R-project.org/package=knitr" class="external-link">knitr</a></em> (Xie,
2014)</li>
<li>
<em><a href="https://CRAN.R-project.org/package=knitrBootstrap" class="external-link">knitrBootstrap</a></em>
(Hester, 2018)</li>
<li><em><a href="https://CRAN.R-project.org/package=mgcv" class="external-link">mgcv</a></em></li>
<li>
<em><a href="https://CRAN.R-project.org/package=pheatmap" class="external-link">pheatmap</a></em>
(Kolde, 2019)</li>
<li>
<em><a href="https://CRAN.R-project.org/package=RColorBrewer" class="external-link">RColorBrewer</a></em>
(Neuwirth, 2022)</li>
<li>
<em><a href="https://CRAN.R-project.org/package=RefManageR" class="external-link">RefManageR</a></em>
(McLean, 2017)</li>
<li>
<em><a href="https://CRAN.R-project.org/package=rmarkdown" class="external-link">rmarkdown</a></em>
(Allaire, Xie, Dervieux et al., 2023)</li>
<li>
<em><a href="https://CRAN.R-project.org/package=sessioninfo" class="external-link">sessioninfo</a></em>
(Wickham, Chang, Flight, Müller, and Hester, 2021)</li>
<li>
<em><a href="https://bioconductor.org/packages/3.17/TxDb.Hsapiens.UCSC.hg19.knownGene" class="external-link">TxDb.Hsapiens.UCSC.hg19.knownGene</a></em>
(Carlson and Maintainer, 2015)</li>
<li><em><a href="https://CRAN.R-project.org/package=whisker" class="external-link">whisker</a></em></li>
</ul>
<p>Code for creating the vignette</p>
<div class="sourceCode" id="cb27"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Create the vignette</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="st"><a href="https://github.com/rstudio/rmarkdown" class="external-link">"rmarkdown"</a></span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/system.time.html" class="external-link">system.time</a></span><span class="op">(</span><span class="fu"><a href="https://pkgs.rstudio.com/rmarkdown/reference/render.html" class="external-link">render</a></span><span class="op">(</span><span class="st">"regionReport.Rmd"</span>, <span class="st">"BiocStyle::html_document"</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## Extract the R code</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="st"><a href="https://yihui.org/knitr/" class="external-link">"knitr"</a></span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/knitr/man/knit.html" class="external-link">knit</a></span><span class="op">(</span><span class="st">"regionReport.Rmd"</span>, tangle <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb28"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Clean up</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/unlink.html" class="external-link">unlink</a></span><span class="op">(</span><span class="st">"derfinderReport-example"</span>, recursive <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div>
<p>Date the vignette was generated.</p>
<pre><code><span><span class="co">## [1] "2023-05-07 01:27:35 UTC"</span></span></code></pre>
<p>Wallclock time spent generating the vignette.</p>
<pre><code><span><span class="co">## Time difference of 15.333 secs</span></span></code></pre>
<p><code>R</code> session information.</p>
<pre><code><span><span class="co">## <span style="color: #00BBBB; font-weight: bold;">─ Session info ───────────────────────────────────────────────────────────────────────────────────────────────────────</span></span></span>
<span><span class="co">##  <span style="color: #555555; font-style: italic;">setting </span> <span style="color: #555555; font-style: italic;">value</span></span></span>
<span><span class="co">##  version  R version 4.3.0 (2023-04-21)</span></span>
<span><span class="co">##  os       Ubuntu 22.04.2 LTS</span></span>
<span><span class="co">##  system   x86_64, linux-gnu</span></span>
<span><span class="co">##  ui       X11</span></span>
<span><span class="co">##  language en</span></span>
<span><span class="co">##  collate  en_US.UTF-8</span></span>
<span><span class="co">##  ctype    en_US.UTF-8</span></span>
<span><span class="co">##  tz       UTC</span></span>
<span><span class="co">##  date     2023-05-07</span></span>
<span><span class="co">##  pandoc   2.19.2 @ /usr/local/bin/ (via rmarkdown)</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## <span style="color: #00BBBB; font-weight: bold;">─ Packages ───────────────────────────────────────────────────────────────────────────────────────────────────────────</span></span></span>
<span><span class="co">##  <span style="color: #555555; font-style: italic;">package             </span> <span style="color: #555555; font-style: italic;">*</span> <span style="color: #555555; font-style: italic;">version  </span> <span style="color: #555555; font-style: italic;">date (UTC)</span> <span style="color: #555555; font-style: italic;">lib</span> <span style="color: #555555; font-style: italic;">source</span></span></span>
<span><span class="co">##  AnnotationDbi          1.62.1    <span style="color: #555555;">2023-05-02</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  backports              1.4.1     <span style="color: #555555;">2021-12-13</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  base64enc              0.1-3     <span style="color: #555555;">2015-07-28</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  bibtex                 0.5.1     <span style="color: #555555;">2023-01-26</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  Biobase                2.60.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  BiocFileCache          2.8.0     <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  BiocGenerics           0.46.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  BiocIO                 1.10.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  BiocManager            1.30.20   <span style="color: #555555;">2023-02-24</span> <span style="color: #555555;">[2]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  BiocParallel           1.34.1    <span style="color: #555555;">2023-05-05</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  BiocStyle            * 2.28.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  biomaRt                2.56.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  Biostrings             2.68.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  bit                    4.0.5     <span style="color: #555555;">2022-11-15</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  bit64                  4.0.5     <span style="color: #555555;">2020-08-30</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  bitops                 1.0-7     <span style="color: #555555;">2021-04-24</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  blob                   1.2.4     <span style="color: #555555;">2023-03-17</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  bookdown               0.33      <span style="color: #555555;">2023-03-06</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  BSgenome               1.68.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  bslib                  0.4.2     <span style="color: #555555;">2022-12-16</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  bumphunter             1.42.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  cachem                 1.0.8     <span style="color: #555555;">2023-05-01</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  checkmate              2.2.0     <span style="color: #555555;">2023-04-27</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  cli                    3.6.1     <span style="color: #555555;">2023-03-23</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  cluster                2.1.4     <span style="color: #555555;">2022-08-22</span> <span style="color: #555555;">[3]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  codetools              0.2-19    <span style="color: #555555;">2023-02-01</span> <span style="color: #555555;">[3]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  colorspace             2.1-0     <span style="color: #555555;">2023-01-23</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  crayon                 1.5.2     <span style="color: #555555;">2022-09-29</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  curl                   5.0.0     <span style="color: #555555;">2023-01-12</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  data.table             1.14.8    <span style="color: #555555;">2023-02-17</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  DBI                    1.1.3     <span style="color: #555555;">2022-06-18</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  dbplyr                 2.3.2     <span style="color: #555555;">2023-03-21</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  DEFormats              1.28.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  DelayedArray           0.26.2    <span style="color: #555555;">2023-05-05</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  derfinder            * 1.34.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  derfinderHelper        1.34.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  desc                   1.4.2     <span style="color: #555555;">2022-09-08</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  DESeq2                 1.40.1    <span style="color: #555555;">2023-05-02</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  digest                 0.6.31    <span style="color: #555555;">2022-12-11</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  doRNG                  1.8.6     <span style="color: #555555;">2023-01-16</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  dplyr                  1.1.2     <span style="color: #555555;">2023-04-20</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  edgeR                  3.42.2    <span style="color: #555555;">2023-05-02</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  evaluate               0.20      <span style="color: #555555;">2023-01-17</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  fansi                  1.0.4     <span style="color: #555555;">2023-01-22</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  fastmap                1.1.1     <span style="color: #555555;">2023-02-24</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  filelock               1.0.2     <span style="color: #555555;">2018-10-05</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  foreach                1.5.2     <span style="color: #555555;">2022-02-02</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  foreign                0.8-84    <span style="color: #555555;">2022-12-06</span> <span style="color: #555555;">[3]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  Formula                1.2-5     <span style="color: #555555;">2023-02-24</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  fs                     1.6.2     <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  generics               0.1.3     <span style="color: #555555;">2022-07-05</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  GenomeInfoDb           1.36.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  GenomeInfoDbData       1.2.10    <span style="color: #555555;">2023-05-07</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  GenomicAlignments      1.36.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  GenomicFeatures        1.52.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  GenomicFiles           1.36.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  GenomicRanges          1.52.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  ggplot2                3.4.2     <span style="color: #555555;">2023-04-03</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  glue                   1.6.2     <span style="color: #555555;">2022-02-24</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  gridExtra              2.3       <span style="color: #555555;">2017-09-09</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  gtable                 0.3.3     <span style="color: #555555;">2023-03-21</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  Hmisc                  5.0-1     <span style="color: #555555;">2023-03-08</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  hms                    1.1.3     <span style="color: #555555;">2023-03-21</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  htmlTable              2.4.1     <span style="color: #555555;">2022-07-07</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  htmltools              0.5.5     <span style="color: #555555;">2023-03-23</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  htmlwidgets            1.6.2     <span style="color: #555555;">2023-03-17</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  httr                   1.4.5     <span style="color: #555555;">2023-02-24</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  IRanges                2.34.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  iterators              1.0.14    <span style="color: #555555;">2022-02-05</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  jquerylib              0.1.4     <span style="color: #555555;">2021-04-26</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  jsonlite               1.8.4     <span style="color: #555555;">2022-12-06</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  KEGGREST               1.40.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  knitr                  1.42      <span style="color: #555555;">2023-01-25</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  knitrBootstrap         1.0.2     <span style="color: #555555;">2018-05-24</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  lattice                0.21-8    <span style="color: #555555;">2023-04-05</span> <span style="color: #555555;">[3]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  lifecycle              1.0.3     <span style="color: #555555;">2022-10-07</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  limma                  3.56.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  locfit                 1.5-9.7   <span style="color: #555555;">2023-01-02</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  lubridate              1.9.2     <span style="color: #555555;">2023-02-10</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  magrittr               2.0.3     <span style="color: #555555;">2022-03-30</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  markdown               1.6       <span style="color: #555555;">2023-04-07</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  Matrix                 1.5-4     <span style="color: #555555;">2023-04-04</span> <span style="color: #555555;">[3]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  MatrixGenerics         1.12.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  matrixStats            0.63.0    <span style="color: #555555;">2022-11-18</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  memoise                2.0.1     <span style="color: #555555;">2021-11-26</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  munsell                0.5.0     <span style="color: #555555;">2018-06-12</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  nnet                   7.3-19    <span style="color: #555555;">2023-05-03</span> <span style="color: #555555;">[3]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  pillar                 1.9.0     <span style="color: #555555;">2023-03-22</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  pkgconfig              2.0.3     <span style="color: #555555;">2019-09-22</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  pkgdown                2.0.7     <span style="color: #555555;">2022-12-14</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  plyr                   1.8.8     <span style="color: #555555;">2022-11-11</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  png                    0.1-8     <span style="color: #555555;">2022-11-29</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  prettyunits            1.1.1     <span style="color: #555555;">2020-01-24</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  progress               1.2.2     <span style="color: #555555;">2019-05-16</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  purrr                  1.0.1     <span style="color: #555555;">2023-01-10</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  qvalue                 2.32.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  R6                     2.5.1     <span style="color: #555555;">2021-08-19</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  ragg                   1.2.5     <span style="color: #555555;">2023-01-12</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  rappdirs               0.3.3     <span style="color: #555555;">2021-01-31</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  Rcpp                   1.0.10    <span style="color: #555555;">2023-01-22</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  RCurl                  1.98-1.12 <span style="color: #555555;">2023-03-27</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  RefManageR           * 1.4.0     <span style="color: #555555;">2022-09-30</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  regionReport         * 1.35.0    <span style="color: #555555;">2023-05-07</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  reshape2               1.4.4     <span style="color: #555555;">2020-04-09</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  restfulr               0.0.15    <span style="color: #555555;">2022-06-16</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  rjson                  0.2.21    <span style="color: #555555;">2022-01-09</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  rlang                  1.1.1     <span style="color: #555555;">2023-04-28</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  rmarkdown              2.21      <span style="color: #555555;">2023-03-26</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  rngtools               1.5.2     <span style="color: #555555;">2021-09-20</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  rpart                  4.1.19    <span style="color: #555555;">2022-10-21</span> <span style="color: #555555;">[3]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  rprojroot              2.0.3     <span style="color: #555555;">2022-04-02</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  Rsamtools              2.16.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  RSQLite                2.3.1     <span style="color: #555555;">2023-04-03</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  rstudioapi             0.14      <span style="color: #555555;">2022-08-22</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  rtracklayer            1.60.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  S4Arrays               1.0.1     <span style="color: #555555;">2023-05-01</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  S4Vectors              0.38.1    <span style="color: #555555;">2023-05-02</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  sass                   0.4.6     <span style="color: #555555;">2023-05-03</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  scales                 1.2.1     <span style="color: #555555;">2022-08-20</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  sessioninfo          * 1.2.2     <span style="color: #555555;">2021-12-06</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  stringi                1.7.12    <span style="color: #555555;">2023-01-11</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  stringr                1.5.0     <span style="color: #555555;">2022-12-02</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  SummarizedExperiment   1.30.1    <span style="color: #555555;">2023-05-01</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  systemfonts            1.0.4     <span style="color: #555555;">2022-02-11</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  textshaping            0.3.6     <span style="color: #555555;">2021-10-13</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  tibble                 3.2.1     <span style="color: #555555;">2023-03-20</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  tidyselect             1.2.0     <span style="color: #555555;">2022-10-10</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">CRAN (R 4.3.0)</span></span></span>
<span><span class="co">##  timechange             0.2.0     <span style="color: #555555;">2023-01-11</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  utf8                   1.2.3     <span style="color: #555555;">2023-01-31</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  VariantAnnotation      1.46.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  vctrs                  0.6.2     <span style="color: #555555;">2023-04-19</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  xfun                   0.39      <span style="color: #555555;">2023-04-20</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  XML                    3.99-0.14 <span style="color: #555555;">2023-03-19</span> <span style="color: #555555;">[1]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  xml2                   1.3.4     <span style="color: #555555;">2023-04-27</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  XVector                0.40.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">##  yaml                   2.3.7     <span style="color: #555555;">2023-01-23</span> <span style="color: #555555;">[2]</span> <span style="color: #BB00BB; font-weight: bold;">RSPM (R 4.3.0)</span></span></span>
<span><span class="co">##  zlibbioc               1.46.0    <span style="color: #555555;">2023-04-25</span> <span style="color: #555555;">[1]</span> <span style="color: #555555;">Bioconductor</span></span></span>
<span><span class="co">## </span></span>
<span><span class="co">## <span style="color: #555555;"> [1] /__w/_temp/Library</span></span></span>
<span><span class="co">## <span style="color: #555555;"> [2] /usr/local/lib/R/site-library</span></span></span>
<span><span class="co">## <span style="color: #555555;"> [3] /usr/local/lib/R/library</span></span></span>
<span><span class="co">## </span></span>
<span><span class="co">## <span style="color: #00BBBB; font-weight: bold;">──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────</span></span></span></code></pre>
</div>
<div class="section level2">
<h2 id="bibliography">Bibliography<a class="anchor" aria-label="anchor" href="#bibliography"></a>
</h2>
<p>This vignette was generated using <em><a href="https://bioconductor.org/packages/3.17/BiocStyle" class="external-link">BiocStyle</a></em>
(Oleś, 2023) with <em><a href="https://CRAN.R-project.org/package=knitr" class="external-link">knitr</a></em> (Xie,
2014) and <em><a href="https://CRAN.R-project.org/package=rmarkdown" class="external-link">rmarkdown</a></em>
(Allaire, Xie, Dervieux et al., 2023) running behind the scenes.</p>
<p>Citations made with <em><a href="https://CRAN.R-project.org/package=RefManageR" class="external-link">RefManageR</a></em>
(McLean, 2017).</p>
<p>
<a id="bib-allaire2023rmarkdown"></a><a href="#cite-allaire2023rmarkdown">[1]</a><cite>
J. Allaire, Y. Xie, C. Dervieux, et al. <em>rmarkdown: Dynamic Documents
for R</em>. R package version 2.21. 2023. URL:
<a href="https://github.com/rstudio/rmarkdown" class="external-link">https://github.com/rstudio/rmarkdown</a>.</cite>
</p>
<p>
<a id="bib-GenomeInfoDb"></a><a href="#cite-GenomeInfoDb">[2]</a><cite>
S. Arora, M. Morgan, M. Carlson, et al. <em>GenomeInfoDb: Utilities for
manipulating chromosome and other ‘seqname’ identifiers</em>. 2017. DOI:
<a href="https://doi.org/10.18129/B9.bioc.GenomeInfoDb" class="external-link">10.18129/B9.bioc.GenomeInfoDb</a>.</cite>
</p>
<p>
<a id="bib-auguie2017gridextra"></a><a href="#cite-auguie2017gridextra">[3]</a><cite>
B. Auguie. <em>gridExtra: Miscellaneous Functions for "Grid"
Graphics</em>. R package version 2.3. 2017. URL:
<a href="https://CRAN.R-project.org/package=gridExtra" class="external-link">https://CRAN.R-project.org/package=gridExtra</a>.</cite>
</p>
<p>
<a id="bib-carlson2015txdb"></a><a href="#cite-carlson2015txdb">[4]</a><cite>
M. Carlson and B. P. Maintainer. <em>TxDb.Hsapiens.UCSC.hg19.knownGene:
Annotation package for TxDb object(s)</em>. R package version 3.2.2.
2015.</cite>
</p>
<p>
<a id="bib-edgeR6"></a><a href="#cite-edgeR6">[5]</a><cite> Y. Chen, A.
T. L. Lun, and G. K. Smyth. “Differential expression analysis of complex
RNA-seq experiments using edgeR”. In: <em>Statistical Analysis of Next
Generation Sequencing Data</em>. Ed. by S. Datta and D. Nettleton. New
York: Springer, 2014, pp. 51-74.</cite>
</p>
<p>
<a id="bib-colladotorres2017derfinderplot"></a><a href="#cite-colladotorres2017derfinderplot">[6]</a><cite>
L. Collado-Torres, A. E. Jaffe, and J. T. Leek. <em>derfinderPlot:
Plotting functions for derfinder</em>. <a href="https://github.com/leekgroup/derfinderPlot" class="external-link uri">https://github.com/leekgroup/derfinderPlot</a> - R package
version 1.34.0. 2017. DOI:
<a href="https://doi.org/10.18129/B9.bioc.derfinderPlot" class="external-link">10.18129/B9.bioc.derfinderPlot</a>.
URL:
<a href="http://www.bioconductor.org/packages/derfinderPlot" class="external-link">http://www.bioconductor.org/packages/derfinderPlot</a>.</cite>
</p>
<p>
<a id="bib-colladotorres2016regionreport"></a><a href="#cite-colladotorres2016regionreport">[7]</a><cite>
L. Collado-Torres, A. E. Jaffe, and J. T. Leek. “regionReport:
Interactive reports for region-level and feature-level genomic analyses
[version2; referees: 2 approved, 1 approved with reservations]”. In:
<em>F1000Research</em> 4 (2016), p. 105. DOI:
<a href="https://doi.org/10.12688/f1000research.6379.2" class="external-link">10.12688/f1000research.6379.2</a>.
URL:
<a href="http://f1000research.com/articles/4-105/v2" class="external-link">http://f1000research.com/articles/4-105/v2</a>.</cite>
</p>
<p>
<a id="bib-colladotorres2017flexible"></a><a href="#cite-colladotorres2017flexible">[8]</a><cite>
L. Collado-Torres, A. Nellore, A. C. Frazee, et al. “Flexible expressed
region analysis for RNA-seq with derfinder”. In: <em>Nucl. Acids
Res.</em> (2017). DOI:
<a href="https://doi.org/10.1093/nar/gkw852" class="external-link">10.1093/nar/gkw852</a>.
URL:
<a href="http://nar.oxfordjournals.org/content/early/2016/09/29/nar.gkw852" class="external-link">http://nar.oxfordjournals.org/content/early/2016/09/29/nar.gkw852</a>.</cite>
</p>
<p>
<a id="bib-hester2018knitrbootstrap"></a><a href="#cite-hester2018knitrbootstrap">[9]</a><cite>
J. Hester. <em>knitrBootstrap: ‘knitr’ Bootstrap Framework</em>. R
package version 1.0.2. 2018. URL:
<a href="https://CRAN.R-project.org/package=knitrBootstrap" class="external-link">https://CRAN.R-project.org/package=knitrBootstrap</a>.</cite>
</p>
<p>
<a id="bib-bumphunter"></a><a href="#cite-bumphunter">[10]</a><cite> A.
E. Jaffe, P. Murakami, H. Lee, et al. “Bump hunting to identify
differentially methylated regions in epigenetic epidemiology studies”.
In: <em>International journal of epidemiology</em> 41.1 (2012),
pp. 200–209. DOI:
<a href="https://doi.org/10.1093/ije/dyr238" class="external-link">10.1093/ije/dyr238</a>.</cite>
</p>
<p>
<a id="bib-kolde2019pheatmap"></a><a href="#cite-kolde2019pheatmap">[11]</a><cite>
R. Kolde. <em>pheatmap: Pretty Heatmaps</em>. R package version 1.0.12.
2019. URL:
<a href="https://CRAN.R-project.org/package=pheatmap" class="external-link">https://CRAN.R-project.org/package=pheatmap</a>.</cite>
</p>
<p>
<a id="bib-lawrence2013software"></a><a href="#cite-lawrence2013software">[12]</a><cite>
M. Lawrence, W. Huber, H. Pagès, et al. “Software for Computing and
Annotating Genomic Ranges”. In: <em>PLoS Computational Biology</em> 9 (8
2013). DOI:
<a href="https://doi.org/10.1371/journal.pcbi.1003118" class="external-link">10.1371/journal.pcbi.1003118</a>.
URL:
<a href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003118%7D." class="external-link"></a><a href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003118" class="external-link uri">http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003118</a>}.
</cite></p>


<p>
<a id="bib-love2014moderated"></a><a href="#cite-love2014moderated">[13]</a><cite>
M. I. Love, W. Huber, and S. Anders. “Moderated estimation of fold
change and dispersion for RNA-seq data with DESeq2”. In: <em>Genome
Biology</em> 15 (12 2014), p. 550. DOI:
<a href="https://doi.org/10.1186/s13059-014-0550-8" class="external-link">10.1186/s13059-014-0550-8</a>.</cite>
</p>
<p>
<a id="bib-mccarthy2012differential"></a><a href="#cite-mccarthy2012differential">[14]</a><cite>
D. J. McCarthy, Y. Chen, and G. K. Smyth. “Differential expression
analysis of multifactor RNA-Seq experiments with respect to biological
variation”. In: <em>Nucleic Acids Research</em> 40.10 (2012),
pp. 4288-4297. DOI:
<a href="https://doi.org/10.1093/nar/gks042" class="external-link">10.1093/nar/gks042</a>.</cite>
</p>
<p>
<a id="bib-mclean2017refmanager"></a><a href="#cite-mclean2017refmanager">[15]</a><cite>
M. W. McLean. “RefManageR: Import and Manage BibTeX and BibLaTeX
References in R”. In: <em>The Journal of Open Source Software</em>
(2017). DOI:
<a href="https://doi.org/10.21105/joss.00338" class="external-link">10.21105/joss.00338</a>.</cite>
</p>
<p>
<a id="bib-neuwirth2022rcolorbrewer"></a><a href="#cite-neuwirth2022rcolorbrewer">[16]</a><cite>
E. Neuwirth. <em>RColorBrewer: ColorBrewer Palettes</em>. R package
version 1.1-3. 2022. URL:
<a href="https://CRAN.R-project.org/package=RColorBrewer" class="external-link">https://CRAN.R-project.org/package=RColorBrewer</a>.</cite>
</p>
<p>
<a id="bib-ole2023biocstyle"></a><a href="#cite-ole2023biocstyle">[17]</a><cite>
A. Oleś. <em>BiocStyle: Standard styles for vignettes and other
Bioconductor documents</em>. R package version 2.28.0. 2023. DOI:
<a href="https://doi.org/10.18129/B9.bioc.BiocStyle" class="external-link">10.18129/B9.bioc.BiocStyle</a>.
URL:
<a href="https://bioconductor.org/packages/BiocStyle" class="external-link">https://bioconductor.org/packages/BiocStyle</a>.</cite>
</p>
<p>
<a id="bib-ole2023deformats"></a><a href="#cite-ole2023deformats">[18]</a><cite>
A. Oleś. <em>DEFormats: Differential gene expression data formats
converter</em>. R package version 1.28.0. 2023. DOI:
<a href="https://doi.org/10.18129/B9.bioc.DEFormats" class="external-link">10.18129/B9.bioc.DEFormats</a>.
URL:
<a href="https://bioconductor.org/packages/DEFormats" class="external-link">https://bioconductor.org/packages/DEFormats</a>.</cite>
</p>
<p>
<a id="bib-2023language"></a><a href="#cite-2023language">[19]</a><cite>
R Core Team. <em>R: A Language and Environment for Statistical
Computing</em>. R Foundation for Statistical Computing. Vienna, Austria,
2023. URL:
<a href="https://www.R-project.org/" class="external-link">https://www.R-project.org/</a>.</cite>
</p>
<p>
<a id="bib-robinson2010edger"></a><a href="#cite-robinson2010edger">[20]</a><cite>
M. D. Robinson, D. J. McCarthy, and G. K. Smyth. “edgeR: a Bioconductor
package for differential expression analysis of digital gene expression
data”. In: <em>Bioinformatics</em> 26.1 (2010), pp. 139-140. DOI:
<a href="https://doi.org/10.1093/bioinformatics/btp616" class="external-link">10.1093/bioinformatics/btp616</a>.</cite>
</p>
<p>
<a id="bib-wickham2016ggplot2"></a><a href="#cite-wickham2016ggplot2">[21]</a><cite>
H. Wickham. <em>ggplot2: Elegant Graphics for Data Analysis</em>.
Springer-Verlag New York, 2016. ISBN: 978-3-319-24277-4. URL:
<a href="https://ggplot2.tidyverse.org" class="external-link">https://ggplot2.tidyverse.org</a>.</cite>
</p>
<p>
<a id="bib-wickham2021sessioninfo"></a><a href="#cite-wickham2021sessioninfo">[22]</a><cite>
H. Wickham, W. Chang, R. Flight, et al. <em>sessioninfo: R Session
Information</em>. <a href="https://github.com/r-lib/sessioninfo#readme" class="external-link uri">https://github.com/r-lib/sessioninfo#readme</a>, <a href="https://r-lib.github.io/sessioninfo/" class="external-link uri">https://r-lib.github.io/sessioninfo/</a>. 2021.</cite>
</p>
<p>
<a id="bib-xie2014knitr"></a><a href="#cite-xie2014knitr">[23]</a><cite>
Y. Xie. “knitr: A Comprehensive Tool for Reproducible Research in R”.
In: <em>Implementing Reproducible Computational Research</em>. Ed. by V.
Stodden, F. Leisch and R. D. Peng. ISBN 978-1466561595. Chapman and
Hall/CRC, 2014.</cite>
</p>
<p>
<a id="bib-xie2023wrapper"></a><a href="#cite-xie2023wrapper">[24]</a><cite>
Y. Xie, J. Cheng, and X. Tan. <em>DT: A Wrapper of the JavaScript
Library ‘DataTables’</em>. R package version 0.27. 2023. URL:
<a href="https://github.com/rstudio/DT" class="external-link">https://github.com/rstudio/DT</a>.</cite>
</p>
<p>
<a id="bib-yin2012ggbio"></a><a href="#cite-yin2012ggbio">[25]</a><cite>
T. Yin, D. Cook, and M. Lawrence. “ggbio: an R package for extending the
grammar of graphics for genomic data”. In: <em>Genome Biology</em> 13.8
(2012), p. R77.</cite>
</p>
<p>
<a id="bib-yin2023biovizbase"></a><a href="#cite-yin2023biovizbase">[26]</a><cite>
T. Yin, M. Lawrence, and D. Cook. <em>biovizBase: Basic graphic
utilities for visualization of genomic data.</em> R package version
1.48.0. 2023. DOI:
<a href="https://doi.org/10.18129/B9.bioc.biovizBase" class="external-link">10.18129/B9.bioc.biovizBase</a>.
URL:
<a href="https://bioconductor.org/packages/biovizBase" class="external-link">https://bioconductor.org/packages/biovizBase</a>.</cite>
</p>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Leonardo Collado-Torres, Andrew E. Jaffe, Jeffrey T. Leek.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
